spec/controllers/organizations_controller_spec.rb

```diff
@@ -5,16 +5,6 @@
   let(:member) { Fabricate(:member, organization: organization) }
   let(:user) { member.user }
 
-  describe 'GET #show' do
-    it 'displays the organization page' do
-      get 'show', id: organization.id
-
-      expect(assigns(:organization)).to eq(organization)
-      expect(response.status).to eq(200)
-      expect(response.body).to include(organization.name)
```

Comment:
I would only remove this line. The rest is actually testing the controller's behavior and we still want it.